packages/datetime/src/dateInput.tsx

```diff
@@ -75,19 +74,19 @@ export interface IDateInputProps extends IDatePickerBaseProps, IProps {
     /**
      * Called when the user finishes typing in a new date and the date causes an error state.
      * If the date is invalid, `new Date(undefined)` will be returned. If the date is out of range,
-     * the out of range date will be returned (`onChange` is not called in this case).
+     * the out-of-range date will be returned (`onChange` is not called in this case).
      */
     onError?: (errorDate: Date) => void;
 
     /**
-     * If true, the Popover will open when the user clicks on the input. If false, the Popover will only
+     * If `true`, the popover will open when the user clicks on the input. If `false`, the popover will only
      * open when the calendar icon is clicked.
      * @default true
      */
     openOnFocus?: boolean;
 
     /**
-     * The error message to display when the date selected is out of range.
+     * The error message to display when the date selected is out-of-range.
```

Comment:
it's kind of like gluten-free: it's hyphenated when used as an adjective. @eparish any thoughts?